src/blocking.rs

```diff
@@ -295,6 +300,53 @@ impl BlockingClient {
         }
     }
 
+	/// Broadcast a package of [`Transaction`] to Esplora
+	///
+	/// if `maxfeerate` is provided, any transaction whose
+	/// fee is higher will be rejected
+	///
+	/// if  `maxburnamount` is provided, any transaction
+	/// with higher provably unspendable outputs amount
+	/// will be rejected
+	pub fn broadcast_pkg(
+		&self,
+		transactions: &[Transaction],
+		maxfeerate: Option<f64>,
+		maxburnamount: Option<f64>)
+	-> Result<SubmitPackageResult, Error> {
+		let serialized_txs = transactions
+			.iter()
+			.map(|tx| serialize(&tx).to_lower_hex_string())
+			.collect::<Vec<_>>();
+
+		let mut request = minreq::post(format!("{}/txs/package", self.url)).with_body(
+			serde_json::to_string(&serialized_txs)
+			.unwrap()
+			.as_bytes()
+			.to_vec()
+		);
+
+		if let Some(maxfeerate) = maxfeerate {
+			request = request.with_param("maxfeerate", maxfeerate.to_string())
+		}
+
+		if let Some(maxburnamount) = maxburnamount {
+			request = request.with_param("maxburnamount", maxburnamount.to_string())
+		}
```

Comment:
Same comment as for the `sync` example.

Please do verify the amounts client-side before sending the request